### PR TITLE
PPA Installation Steps

### PR DESCRIPTION
**Pull request by orschiro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,19 +59,12 @@ On Windows you can install [Chocolatey package](https://chocolatey.org/packages/
...
-```
-
-Install the app from Ubuntu Sotware Center or from command line.
-
-```bash
-apt update
-apt install copyq
+$ sudo add-apt-repository ppa:hluk/copyq
+$ sudo apt get update
+$ sudo apt-get install
```
Comment (hluk) [inline README.md]: The last two line probably should be:
```bash
sudo apt update
sudo apt install copyq
```

**Comment by orschiro** [comment]
Is one preferred over the other?

**Comment by hluk** [comment]
@orschiro `apt` is modern version of `apt-get`, is easier to type and should be more user-friendly. Though if you want to use `apt-get`, it's OK, but the command is still missing one dash and `install` command seems to be missing one argument.

**Comment by orschiro** [comment]
You are completely right. 

Sorry for this mistake. 

I corrected this by another commit because I didn't know how to edit my previous commit using the Github interface. Do you know?

**Comment by hluk** [comment]
@orschiro On GitLab you would normally push new commits when doing PR (not sure if there is a standard way to follow on github). So this is OK.

**Comment by extrealm** [comment]
Hi Hluk, just fyi, now 14.04 ppa work like charm.

**Pull request merged**